src/components/features/ShopFurniture/ShopFurniture.module.scss

```
@@ -1,6 +1,9 @@
 @import "../../../styles/settings.scss";
 
 .root {
+  .icon {
+    margin-right: 10px;
```

```suggestion
    margin-right: 10px;
    
    &:first-of-type {
      color: $newFurniture-activeIcon-color;
    }
```
Later on `:first-of-type` could become `.active` or `.activeIcon `and would be added with some js method or a shorthand `if` within `className= property`. Also you need to add `$newFurniture-activeIcon-color: $color-tigersEye;` in `style/settings.scss` in the `// new furniture` section of this file. Btw. entire section has a typo in variable: it's $new**Furiniture** and should be $new**Furniture** 😉